### DISCUSSION TITLE
setting mask token with ESM C API

### DISCUSSION THREAD

**Discussion by younggon0** [evolutionaryscale/esm#218] [Q&A]
I'm planning to use ESM-C for single mutant variant scoring. My approach would be to use the common method of masking one token and utilizing the logits for that position. Since I'm working with the 6B parameter version that's only available through the API, is it possible to set a mask token using the API interface?
